packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -0,0 +1,82 @@
+import { readdir } from "fs";
+import path, { dirname } from "path";
+import vscode from "vscode";
+import { Executable } from "vscode-languageclient/node.js";
+import { StartFileName } from "./const.js";
+import logger from "./log/logger.js";
+import { isFile, loadModule, normalizeSlash } from "./utils.js";
+
+export const toOutput = (str: string) => {
+  str
+    .trim()
+    .split("\n")
+    .forEach((line) => logger.info(line));
+};
+export const toError = (str: string) => {
```

Comment:
toOutput and toError are confusing here as utils. it's hard to figure out how to use them by just looking at the name. I guess you can use the spawnExecutionAndLogToOutput method in utils directly? so you don't need them here?